src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -165,6 +174,11 @@ public void onBuildStart(AbstractBuild<?, ?> build,
         } else {
             sb.append(Ghprb.replaceMacros(build, listener, startedStatus));
         }
+
+        if(showMatrixStatus && !(build.getProject() instanceof MatrixProject)){
```

Comment:
My concern is for a regular project that had nothing to do with a Matrix job.  If my job is running it will also not be a matrix job like the child of a matrix job.

For example I create a Freestyle project and don't check the box.  Then I will never see a status.